apps/vscode.py

```diff
@@ -55,44 +55,17 @@ def select_lines_function(m):
 
 context.keymap(
     {
-        # Selecting text
```

Comment:
did you mean to remove all of these commands?

Reply:
Yeah. It seemed like there was a lot of overlap with generic_editor.py. Was that a bad move?